actions/sequester/Quest2GitHub.Tests/GetIterationTests.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Quest2GitHub.Models;
+
+namespace Quest2GitHub.Tests;
+public class GetIterationTests
```

Comment:
This was all tests to make sure I was parsing the iteration objects correctly to compare Quest iterations with GitHub iterations.

They were, but leaving the tests for the future.